src/components/apps/launch/ResourceRequirements.js

```diff
@@ -0,0 +1,360 @@
+                                            <ResourceRequirementsHeader
+                                                headerMessageKey="resourceRequirementsForStep"
+                                                step={reqs.step_number + 1}
+                                            />
+                                        </Typography>
+                                    </ExpansionPanelSummary>
+                                    <ExpansionPanelDetails
+                                        className={
+                                            classes.expansionPanelDetails
+                                        }
+                                    >
+                                        <StepResourceRequirementsForm
+                                            baseId={buildDebugId(
+                                                baseId,
+                                                reqs.step_number
+                                            )}
+                                            requirements={reqs}
+                                            index={index}
+                                            defaultMaxCPUCores={
+                                                defaultMaxCPUCores
+                                            }
+                                            defaultMaxMemory={defaultMaxMemory}
+                                            defaultMaxDiskSpace={
+                                                defaultMaxDiskSpace
+                                            }
+                                        />
+                                    </ExpansionPanelDetails>
+                                </ExpansionPanel>
+                            );
+                        })
+                    )}
+                </ExpansionPanelDetails>
+            </ExpansionPanel>
+        );
+    },
+    messages
+);
+
+const StepResourceRequirementsReview = ({
+    baseId,
+    stepRequirements,
+    headerMessageKey,
+}) => {
+    const {
+        step_number,
+        min_cpu_cores,
+        min_memory_limit,
+        min_disk_space,
+    } = stepRequirements;
+
+    return (
+        !!(min_cpu_cores || min_memory_limit || min_disk_space) && (
+            <ExpansionPanel defaultExpanded>
+                <ExpansionPanelSummary
+                    expandIcon={
+                        <ExpandMore
+                            id={buildDebugId(
+                                baseId,
+                                step_number,
+                                ids.BUTTONS.EXPAND
+                            )}
+                        />
+                    }
+                >
+                    <Typography variant="subtitle1">
+                        <ResourceRequirementsHeader
+                            headerMessageKey={headerMessageKey}
+                            step={step_number + 1}
+                        />
+                    </Typography>
```

Comment:
I think i had previously requested this feature in DE as well. Is it possible to add the step name here ? An end user coming to DE just to use an `app` will have no understanding of multi-step/single step app.

Reply:
It will be possible with an update to the app description endpoint, or I can look for another endpoint we can call to get this info.